examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -3585,12 +3605,32 @@ endpoint 1 {
   }
 
   server cluster WindowCovering {
-    ram      attribute type;
+    ram      attribute type default = 0x08;
+    ram      attribute physicalClosedLimitLift default = 0xFFFF;
```

Comment:
This does not look right.  The minimal app is supposed to disable all optional functionality and enable as few features as possible while still being spec-compliant, and these bits are optional.  Please make sure only the things that are mandatory (given the Lift feature is enabled) are enabled here.

Reply:
Ah ok i wasn't aware of this , i just wanted to aligned it , i ll remove